### ISSUE TITLE
Can't save large simulation data w/o -v7.3 flag | Is there a good reason `sim_data`'s -v7.3 save flag is commented out?

### ISSUE THREAD

**Issue by asoplata** [jsherfey/dnsim#58]
So, I just noticed that none of my larger simulations (e.g. 100 cells x 2 cell types, for anything over 3 seconds at `dt` of 0.01 milliseconds) actually save to `sim_data`, probably due to this very simple error:
"Warning: Variable 'sim_data' cannot be saved to a MAT-file whose version is older than 7.3. To save this variable, use the -v7.3 switch. Skipping..." as discussed in [this MATLAB Central thread](http://www.mathworks.com/matlabcentral/answers/15521-matlab-function-save-and-v7-3). Never mind the fact that I question why such a thing is a warning instead of an error, in `matlab/functions/biosimdriver.m` [there's (Line 81)](https://github.com/jsherfey/dnsim/blob/dev/matlab/functions/biosimdriver.m#L81) a [few places (Line 143)](https://github.com/jsherfey/dnsim/blob/dev/matlab/functions/biosimdriver.m#L143) where an optional `-v7.3` flag was there previously, but has been commented out in the current version of the `dev` branch for a while. Testing putting that flag parameter in the main `sim_data` save line (81) solves this problem, probably for the reasons mentioned in the thread (a variable above a certain size, near 2GB, MUST be saved into a version 7.3 MAT-file).

**tl;dr** I'm marking this as a "bug" because it's something that needs fixing as opposed to what's in the `dev` branch right now, but I'm mainly making this an issue in order to ask, **are there any good reasons why NOT to add this flag to all DNSIM saves of `sim_data`?**

**Comment by jsherfey** [comment]
It's probably a good idea to add the flag to support saving large data
files. I don't know any reason why that would be a problem.
